en/publish/publish-wechatgame-sub-domain.md

```diff
@@ -140,32 +140,32 @@ Since the code and resources of the WeChat open data context cannot be shared wi
 
 ## Release Steps
 
-1. Open the main context project, open the build release panel in **Menu Bar --> Project**, select the **Wechat Game** platform, and fill in the **Open Data Field Code Directory**. This directory is the path where the open data context is built, and this path needs to be placed in the main context build directory. Then click on **Build**.
+1. Open the main context project, open the **Build panel** in **Menu Bar --> Project**, select the **WeChat Game** platform, and fill in the **Open Data Context Root**. This directory is the path to the publishing package that is generated after the open data context is built. Then click on **Build**.
 
 ![](./publish-wechatgame/maintest-build.png)
 
-This step will automatically configure the user to the main context project **build -> wechatgame -> game.json** to identify the directory where the open data context file is located under the main context distribution package.
+This step will help the user automatically configure the Open Data Context Root into the main context project **build -> wechatgame -> game.json** to identify the directory where the open data context file is located under the main context distribution package.
 
 ![](./publish-wechatgame/game-json.png)
 
-2. Open the open data context project, open the build release panel, select the **Wechat Game Open Data Context** platform.
+2. Open the open data context project, open the **Build panel**, select the **WeChat Game Open Data Context** platform.
 
-3. The **release path ** set the same path in the **Open Data context code directory** filled in the main context, that is, assigned to the release package directory of the main context project. Then click on **Build**.
+3. The **Build path ** set the same path in the **Open Data Context Root** filled in the main context, that is, assigned to the release package directory of the main context project. Then click on **Build**.
```

Comment:
**Build path ** -> **Build** path

Reply:
fixed and added a regex to my list of things to check for each document.